layouts/_default/single.html

```diff
@@ -30,6 +30,11 @@ <h1 class="post-title">
   <div class="post-content">
     {{ .Content }}
   </div>
+  {{ if $.Site.Params.showLastUpdated }}
```

Comment:
Are you sure you want this at the end of a post? We could align this with a creation date.

Just asking ;-)